plugins/mcreator-localization/lang/texts.properties

```diff
@@ -899,6 +899,7 @@ blockly.block.entity_checkgamemode=Is %1 in game mode
 blockly.block.entity_clear_inventory=Clear inventory of %1
 blockly.block.entity_clearpotions=Clear all potion effects of %1
 blockly.block.entity_close_gui=Close any GUI open for %1
+blockly.block.entity_create=%1 Create new %2 instance without spawning
```

Comment:
`%1 Create new %2 instance without spawning and animations`

Reply:
Maybe `...static...`?

Reply:
static may be hard to understand for non-programming users and could at the same time confuse programming users

without spawning and motion?

Reply:
How about `%1 Create new motionless instance of type %2 without spawning` ?